pkg/cmd/kafka/acl/admin/admin.go

```diff
@@ -83,7 +83,7 @@ func NewAdminACLCommand(f *factory.Factory) *cobra.Command {
 			}
 
 			// user and service account should not allow wildcard
-			if userID == aclutil.Wildcard || serviceAccount == aclutil.Wildcard {
+			if userID == aclutil.Wildcard || serviceAccount == aclutil.Wildcard || userID == aclutil.AllAlias || serviceAccount == aclutil.AllAlias {
```

Comment:
How about `if aclutil.GetResourceName(userID) == aclutil.Wildcard`

Reply:
I don't think it will be a good idea to use `GetResourceName` at places that don't support alias.